server/leafnode.go

```diff
@@ -245,15 +260,15 @@ func (s *Server) leafNodeAcceptLoop(ch chan struct{}) {
 		net.JoinHostPort(opts.LeafNode.Host, strconv.Itoa(l.Addr().(*net.TCPAddr).Port)))
 
 	s.mu.Lock()
-	tlsReq := opts.LeafNode.TLSConfig != nil
-	tlsVerify := tlsReq && opts.LeafNode.TLSConfig.ClientAuth == tls.RequireAndVerifyClientCert
+	tlsRequired := opts.LeafNode.TLSConfig != nil
```

Comment:
Just realized that we should add the same warning than for routes and gateways regarding use of "insecure" in LN tls config...

Reply:
Probably right. Separate PR or want me to update this one?

Reply:
I will do separate one.